app/helpers/devise_helper.rb

```diff
@@ -0,0 +1,15 @@
+module DeviseHelper
+  def devise_error_messages!
```

Comment:
Not sure.. I was following this: https://github.com/plataformatec/devise/wiki/How-To:-Integrate-I18n-Flash-Messages-with-Devise-and-Bootstrap